helm/superset/values.yaml

```diff
@@ -35,7 +35,7 @@ bootstrapScript: |
   rm -rf /var/lib/apt/lists/* && \
   pip install \
     psycopg2-binary==2.9.1 \
-    redis==3.5.3 && \
+    redis==4.3.4 && \
```

Comment:
our current base requirements is: https://github.com/apache/superset/blob/master/requirements/base.txt#L242